### PR TITLE
Added screenshot to README

### PR DESCRIPTION
**Pull request by Sudoxo**
Added header and screenshot to README.md file [GCI task]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Moe
+
+![Moe screenshot](Moe.png "Moe")
+
 Moe is a program to stay cute mascot on active window. How about mind cooler for your desktop?
```
Comment (humdingerb) [inline README.md]: Can you improve the wording here, please?
If you don't understand it all either, come up with something nice... :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
 
 Check the [HTML documentation](./doc/en/index.html).
```
Comment (humdingerb) [inline README.md]: Please add `http://htmlpreview.github.com/?` in front of the complete URL to that html page to get a nice HTML preview of it.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Review by Sudoxo — COMMENTED** [review]
Done

File: README.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 ![Moe screenshot](Moe.png "Moe")
 
-Moe is a program to stay cute mascot on active window. How about mind cooler for your desktop?
+Moe is a program which attaches a character to the active window.
 
-Check the [HTML documentation](./doc/en/index.html).
+[HTML documentation](http://htmlpreview.github.com/doc/en/index.html)
```
Comment (humdingerb) [inline README.md]: You have to use the whole, expanded URL, i.e. where you land when you click the original link, then add the "htmlpreview" part:
`http://htmlpreview.github.com/?https://github.com/HaikuArchives/Moe/blob/master/doc/en/index.html`

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks.

**Pull request merged**